xonsh/platform.py

```diff
@@ -210,13 +232,13 @@ def scandir(path):
     BASH_COMPLETIONS_DEFAULT = (
         '/usr/local/etc/bash_completion',
         '/opt/local/etc/profile.d/bash_completion.sh')
-elif ON_WINDOWS:
-    progamfiles = os.environ.get('PROGRAMFILES', 'C:/Program Files')
+elif ON_WINDOWS and GIT_FOR_WINDOWS_PATH:
```

Comment:
so if `GIT_FOR_WINDOWS_PATH` is None, then `BASH_COMPLETIONS_DEFAULT` is an empty tuple?  (Just asking because I don't know how it was working/not working before and want to check)